resync/api/monitoring_dashboard.py

```
@@ -178,24 +295,25 @@ async def add_error_sample(self, error: Exception) -> None:
+            new_alerts = self._compute_alerts(sample)
+
+            pipe = redis.pipeline()
+            pipe.lpush(REDIS_KEY_HISTORY, data)
+            pipe.ltrim(REDIS_KEY_HISTORY, 0, MAX_SAMPLES - 1)
+            pipe.set(REDIS_KEY_LATEST, data)
+            for alert in new_alerts:
+                pipe.lpush(REDIS_KEY_ALERTS, json_dumps(alert))
+            if new_alerts:
+                pipe.ltrim(REDIS_KEY_ALERTS, 0, 19)
             await pipe.execute()
         except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)